src/main/java/com/google/step/finscholar/data/DemographicCategories.java

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.finscholar.data;
+
+/** The class with string constants about demographic categories. */ 
+public class DemographicCategories {
+
+    public static final String HISPANIC = "Hispanic";
+    public static final String WHITE = "White";
+    public static final String BLACK_OR_AFRICAN_AMERICAN = "Black or African American";
+    public static final String AMERICAN_INDIAN = "American Indian";
+    public static final String ALASKA_NATIVE = "Alaska Native";
+    public static final String ASIAN = "Asian";
+    public static final String PACIFIC_ISLANDER = "Native Hawaiian or Other Pacific Islander";
+  
+    public static final String INTERNATIONAL = "international student";
```

Comment:
I'd break this into two enums:
Ethnicity
Gender

And these three into booleans.

Reply:
I moved ethnicities and gender into enums. And I think later all data will be directly retrieved from database, so I can just make uncommon fields to be string at this point.